index.html

```diff
@@ -4,7 +4,7 @@
     <head>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>Portfolio</title>
+        <title>Sanja's Portfolio</title>
```

Comment:
- [ ] Kindly change the contrast to be at least `4.5` for these failed elements because they failed in the contract check
![image](https://user-images.githubusercontent.com/24703171/166477092-fd535016-fd69-4631-80f7-d75380909e07.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)